extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -4,11 +4,13 @@
   <[$ctrl.warningText]>
 </div>
 <!--
-  Forced to use !important here as it makes the code easy to understand, but does not align with design principles.
-  A solution could be added using CSS, but for some reason the styles defined in this file seem to be overriden by Guppy.
-  Only way to fix this is to override the specificity using !important and force the component to take the styles from here
-  and not the styles defined by Guppy. It seems that even customizing the guppy active or guppy inactive classes has no
-  effect on the styles of the textbox. Again, !important is required to break the specificity for height.
+  Guppy enforces the 'white-space','height' and 'width' styles causing the non-responsive behaviour.
```

Comment:
```suggestion
  Guppy enforces the 'white-space', 'height' and 'width' styles causing the non-responsive behaviour.
```
Also, can you make sure that the lines are 80 characters long at max?